src/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp

```diff
@@ -296,94 +300,158 @@ class boss_shade_of_aran : public CreatureScript
                     }
                 }
                 NormalCastTimer = 1000;
-            } else NormalCastTimer -= diff;
+            }
+            else NormalCastTimer -= diff;
```

Comment:
better placing the body of the ``else`` in a new line, for example:

```
else
    NormalCastTimer -= diff;
```

(as well as in other parts of this PR)